### PR TITLE
mentoring: Add label in OWNERS

### PR DESCRIPTION
**Pull request by nikhita**
/assign @parispittman

### CODE REVIEW

File: mentoring/OWNERS [hunk 1]
```diff
@@ -5,3 +5,4 @@ approvers:
   - sig-contributor-experience-leads
 labels:
   - sig/contributor-experience
+  - area/mentorship-planning
```
Comment (nikhita) [inline mentoring/OWNERS]: Should this label be renamed as `area/mentoring`?

`mentorship/planning` sounds good for issues but not for PRs.

**Comment by parispittman** [comment]
/lgtm
/approve

thank you for these details!!

**Comment by nikhita** [comment]
@parispittman wow that was fast! :D

File: mentoring/OWNERS (hunk 1 above)
Comment (parispittman) [reply mentoring/OWNERS]: I wanted it to stand out that this was the planning/strategy and not for people looking for mentors/mentorship opportunities.

Comment (nikhita) [reply mentoring/OWNERS]: @parispittman wdyt about this? looks like my comment and your lgtm raced :)

Comment (nikhita) [reply mentoring/OWNERS]: > I wanted it to stand out that this was the planning/strategy and not for people looking for mentors/mentorship opportunities.

Ack.

/hold cancel

**Pull request merged**